src/wfuzz/ui/console/mvc.py

```diff
@@ -288,8 +301,20 @@ def result(self, res):
             for i in range(self.printed_lines):
                 sys.stdout.write("\n\r")
 
-    def footer(self, summary):
-        self.term.erase_lines(self.printed_lines + 1)
+        # print progress
+        self._print_progress(res, summary)
+
+    @staticmethod
+    def _print_progress(res, summary):
+        highlight_color = "\033[1;32;40m"
+        normal_text_color = "\033[0;37;40m"
+
+        if summary and res.nres:
+            progress = (res.nres / summary.total_req) * 100
+            sys.stdout.write("\n" + highlight_color + f"[ {round(progress, 2)}% ]" + normal_text_color)
```

Comment:
What about making "the progress string" part of FuzzStats, so you can make something like summary.to_progress_str()